apps/emqx_durable_storage/src/emqx_ds_cache_coordinator.erl

```diff
@@ -0,0 +1,335 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
```

Comment:
Please add Apache license header here

Reply:
Sure, I'll add it.  

As a side note, a few DS modules also don't have this header (bitmask_keymapper is one example).